### ISSUE TITLE
[FEATURE]: Non-sequential heading elements

### ISSUE THREAD

**Issue by irisvw** [fdnd-agency/nebulaxplorer#263]
## Description
The mission page has non-sequential heading elements.

<img width="222" height="163" alt="Image" src="https://github.com/user-attachments/assets/62504780-e5b4-4a96-95ed-87c2ecb4eea0" />

## Solution
The heading elements should be adjusted so that no headings are skipped. In case the page uses components, we should look into a way to adjust the heading level used based on context.

**Comment by irisvw** [comment]
This was fixed in #350 as part of a larger refactoring of the mission page.
